api.php

```diff
@@ -12,6 +12,10 @@
         : db_getJadual());
 }
 
+if(isset($_GET['getfaculty'])) {
```

Comment:
icress has separated campuses and faculty(Selangor) from their API. So, we need to query it separately